### PR TITLE
Replace waitress with gunicorn

### PR DESCRIPTION
**Pull request by vrajmohan**

### CODE REVIEW

File: manifest.base.yml [hunk 1]
```diff
@@ -1,13 +1,14 @@
 ---
 buildpack: python_buildpack
-command: python manage.py migrate --fake-initial && waitress-serve --url-prefix /regulations --port=$VCAP_APP_PORT fec_eregs.wsgi:application
+command: python manage.py migrate --fake-initial && gunicorn fec_eregs.wsgi:application
```
Comment (adborden) [inline manifest.base.yml]: How does gunicorn know what port to use?

Comment (vrajmohan) [inline manifest.base.yml]: I mistakenly followed suit with ATF eRegs, EPA notice-and-comment, etc in not explicitly specifying the port and letting it default to 8000. Your comment made me realize that it is probably an incorrect assumption and I'll push a new commit making the app explicitly use the port specified by the CloudFoundry runtime.

Comment (cmc333333) [inline manifest.base.yml]: I believe gunicorn will pick up the host and port from the environment variables automatically, but it's always a good idea to be explicit. I don't think cloud.gov generally assigns port 8000.

Comment (cmc333333) [inline manifest.base.yml]: Just the defaults so far, but if you find some more optimal configs, we'd carry them over to the other projects.

**Pull request merged**